package.json

```diff
@@ -22,7 +22,17 @@
   },
   "homepage": "https://github.com/manuelam20/workshop-api-testing-js#readme",
   "devDependencies": {
+    "@types/mocha": "^5.2.6",
     "chai": "^4.2.0",
```

Comment:
This is required for the project (test execution) this should be a dependency instead of dev dependency